src/events/render-advert.ts

```diff
@@ -214,7 +215,9 @@ const renderAdvert = (
 					: Promise.resolve();
 
 			return callSizeCallback()
-				.then(() => renderAdvertLabel(advert.node))
+				.then(() =>
+					renderAdvertLabel(advert.node, creativeTemplateId ?? 0),
```

Comment:
We also call `renderAdvertLabel` in the interscroller code, in `background.ts`, so the parameter will need to be added there too